bundle/language_en.properties

```diff
@@ -662,6 +664,19 @@ angal.hospital.visitduration.txt
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.lab.allnegative.txt                                                                              = All negative.
+angal.inventory.close                                                                                  = Close
+angal.inventory.delete                                                                                 = Delete inventory
+angal.inventory.managementwardtitle                                                                    = Manage Ward Inventory
+angal.inventory.new                                                                                    = New inventory
+angal.inventory.referenceshow                                                                          = Reference
+angal.inventory.state                                                                                  = State
+angal.inventory.state.canceled                                                                         = Canceled
+angal.inventory.state.inprogress                                                                       = In progress
+angal.inventory.state.validate                                                                         = Validate
+angal.inventory.update                                                                                 = Update inventory
+angal.inventory.user                                                                                   = User
+angal.inventory.view                                                                                   = View inventory
+angal.inventory.ward                                                                                   = Ward
```

Comment:
Please see https://openhospital.atlassian.net/wiki/spaces/OH/pages/2203779073/Open+Hospital+1.x+Bundles+translations+files for proper formatting of the keys (specifically the suffixes used)